include/aikido/planner/vectorfield/VectorFieldPlanner.hpp

```diff
@@ -109,6 +114,23 @@ aikido::trajectory::UniqueInterpolatedPtr planToEndEffectorPose(
     std::chrono::duration<double> timelimit,
     planner::Planner::Result* result = nullptr);
 
+// std::unique_ptr<aikido::trajectory::Spline> planWithEndEffectorTwist(
+aikido::trajectory::UniqueInterpolatedPtr planWithEndEffectorTwist(
+    const aikido::statespace::dart::ConstMetaSkeletonStateSpacePtr& stateSpace,
+    const statespace::dart::MetaSkeletonStateSpace::State& startState,
+    ::dart::dynamics::MetaSkeletonPtr metaskeleton,
+    const ::dart::dynamics::ConstBodyNodePtr& bn,
+    const Eigen::Vector6d& twistSeq,
```

Comment:
Let's use a full variable name, e.g. twistSequence? (I'm actually not sure what `twistSeq` stands for.)

Reply:
I just followed the convention in `MoveEndEffectorTwistVectorField.cpp`.  I think twistSeq is just simply a twist msg(same in ROS), first 3 elements: angular velocity, last 3 elements: linear velocity.